CONTRIBUTING.md

```diff
@@ -53,11 +53,10 @@ For features that you have a proposed solution for, please write an [RFC](./rfcs
 
 When submitting a pull request, please ensure your branch meets the following criteria:
 
-* It is a passing build (`gulp test --build` returns success).
+* It is a passing build. https://github.com/Sage/carbon/blob/master/README.md#testing
```

Comment:
`https://github.com/Sage/carbon/blob/master/README.md#testing` doesn't explain how to run the tests.